web/crux-ui/src/components/products/versions/images/edit-image-config.tsx

```diff
@@ -1,7 +1,7 @@
 import { IMAGE_WS_REQUEST_DELAY } from '@app/const'
 import { DyoInput } from '@app/elements/dyo-input'
 import { useThrottling } from '@app/hooks/use-throttleing'
-import { ContainerConfig, Environment } from '@app/models'
+import { ContainerConfig, Environment } from '@app/models-config'
```

Comment:
we should make a models folder instead later on. i am fine with this now, but please put a techdept comment into the file's header

Reply:
And create a ticket for it pleaseeeeee 😅